### ISSUE TITLE
Add :focus feature for home page blur blocks

### ISSUE THREAD

**Issue by wenqili** [wenqili/adjacent#6]
Tab is not working on home page

**Comment by avaseghi** [comment]
Specifically, you're not able to tab through the menu and previous issue items. Also, when you tab through the articles you are unable to see the text that is highlighted unless you hover over.  The table of contents also cannot be tabbed.

**Comment by wenqili** [comment]
fixed by adding :focus pseudo class
